notebooks/demo_generate_prsn.ipynb

```diff
@@ -15,17 +15,25 @@
    "source": [
     "from birdy import WPSClient\n",
     "import requests\n",
-    "from pkg_resources import resource_filename\n",
-    "\n",
-    "import os\n",
-    "os.chdir('../')"
+    "from pkg_resources import resource_filename"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "import os \n",
+    "if os.path.basename(os.getcwd()) != \"thunderbird\": # Ensure current directory is always 'thunderbird' for 'resource_filename' to work\n",
+    "    os.chdir('../')"
```

Comment:
This does resolve the current directory issue. Nice work!

Reply:
Could you actually try running `demo_processes.ipynb` again? I pushed a commit that moves the individual demos to their own subdirectory and I'm not sure if I'll have to move up one more level as a result.